features/step_definitions/_constants.rb

```diff
@@ -1,11 +1,11 @@
 OPT_SYM = '(?: :(\S+))?'
 SYM = ':(\S+)'
-OPT_QUOTED = '(?: "(.+?)")?'
-QUOTED = '"(.+?)"'
+OPT_QUOTED = '(?: "([^"\n]+)")?'
+QUOTED = '"([^"\n]+)"'
```

Comment:
we can't have `\n` anyway. The `"` character being forbidden will prevent us from using expressions between `<%= some code with quotes %>`

Reply:
`.` doesn't match `\n`, but negated classes can match `\n`.

I don't think we can have quotes within quotes either.

```ruby
>> '"asdf"'.match('"(.+?)"')
=> #<MatchData "\"asdf\"" 1:"asdf">
>> '"as"d"ff"'.match('"(.+?)"')
=> #<MatchData "\"as\"" 1:"as">
>> '"asdf"'.match('"([^"\n]+)"')
=> #<MatchData "\"asdf\"" 1:"asdf">
>> '"as"d"f"'.match('"([^"\n]+)"')
=> #<MatchData "\"as\"" 1:"as">
```

Reply:
About quotes, see my reply about slashes. wrt `\n` it can match depending on RE flags. In either case, I believe gherkin syntax will not allow new lines anyway so I think the handling of `\n` is just making REs harder to read.

Do you see any place where you can insert `\n` where we don't want it to match?